src/sys/mod.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(all(target_os = "linux",
+          any(target_arch = "x86", target_arch = "x86_64")))]
```

Comment:
Hm I saw [this comment](https://github.com/rust-lang/libc/pull/196#issuecomment-189918276) on the libc PR where added the struct. Is it not possible to add more platforms?